docs/release-notes/config-policies.rst

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+**New Features**
+
+- Cluster: Using config policies, administrators can now set limits on how users can define workloads (e.g.,
+  experiments, notebooks, tensorboards, shells, and commands). Admins may define two types of configurations\:
+  - **Invariant Configs for Experiments**: Settings applied to all experiments within a scope (global or workspace). 
```

Comment:
This spacing is messed up and I cannot figure out why.

Reply:
`**New Features**

-  Cluster: Add :ref:`config policies <config-policies>` to enable administrators to set limits on
   how users can define workloads (e.g., experiments, notebooks, TensorBoards, shells, and
   commands). Administrators can define two types of configurations:

   -  **Invariant Configs for Experiments**: Settings applied to all experiments within a specific
      scope (global or workspace). Invariant configs for other tasks (e.g. notebooks, TensorBoards,
      shells, and commands) are not yet supported.

   -  **Constraints**: Restrictions that prevent users from exceeding resource limits within a
      scope. Constraints can be set independently for experiments and tasks.
`